run_python_tests.py

```diff
@@ -0,0 +1,28 @@
+# run_tests.py
```

Comment:
I think the `unittest` command can handle this on it's own. If that's the case, we won't need this file.